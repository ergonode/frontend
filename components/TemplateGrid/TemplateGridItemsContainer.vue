/*
 * Copyright Â© Bold Brand Commerce Sp. z o.o. All rights reserved.
 * See LICENSE for license details.
 */
<template>
    <div
        class="grid-items-container">
        <slot />
    </div>
</template>

<script>
export default {
    name: 'TemplateGridItemsContainer',
};
</script>

<style lang="scss" scoped>
    .grid-items-container {
        position: absolute;
        top: 0;
        z-index: 10;
        display: grid;
        width: 100%;
        height: auto;
    }
</style>
