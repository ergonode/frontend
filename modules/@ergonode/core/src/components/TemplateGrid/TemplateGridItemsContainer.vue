/*
 * Copyright Â© Bold Brand Commerce Sp. z o.o. All rights reserved.
 * See LICENSE for license details.
 */
<template>
    <div class="grid-items-container">
        <slot />
    </div>
</template>

<script>
export default {
    name: 'TemplateGridItemsContainer',
};
</script>

<style lang="scss" scoped>
    .grid-items-container {
        display: grid;
        height: auto;
    }
</style>
