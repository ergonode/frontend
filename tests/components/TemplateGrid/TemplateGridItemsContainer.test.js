/*
 * Copyright Â© Bold Brand Commerce Sp. z o.o. All rights reserved.
 * See LICENSE for license details.
 */
import { shallowMount } from '@vue/test-utils';
import TemplateGridItemsContainer from '~/components/TemplateGrid/TemplateGridItemsContainer';

describe('TemplateGrid/TemplateGridItemsContainer', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallowMount(TemplateGridItemsContainer);
    });

    it('Component is named well', () => {
        expect(typeof TemplateGridItemsContainer.name).toBe('string');
        expect(TemplateGridItemsContainer.name).toEqual('TemplateGridItemsContainer');
    });
    it('Component has main class', () => {
        expect(wrapper.contains('.grid-items-container')).toBeTruthy();
        expect(wrapper.vm.$el).toMatchSnapshot();
    });
});
