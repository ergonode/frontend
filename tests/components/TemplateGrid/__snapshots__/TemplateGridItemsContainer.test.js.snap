// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TemplateGrid/TemplateGridItemsContainer Component has main class 1`] = `
<div
  class="grid-items-container"
/>
`;
